.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
+  "parser": "babel-eslint",
+  "rules": {
+    "strict": 0,
+    "quotes": [2, "single"]
```

Comment:
JSCS already checks for quotes. Does ESlint make it so we no longer need JSCS? Or should we keep both and only use JSCS for code style preferences, and ESlint for functionality-based preferences?